math/casadi/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0   
+PortGroup           cmake 1.0
```

Comment:
Please use/test version 1.1 of the cmake portgroup

Reply:
Done